PowerEditor/src/NppIO.cpp

```diff
@@ -302,8 +305,14 @@ BufferID Notepad_plus::doOpen(const generic_string& fileName, bool isRecursive,
 				str2display += longFileName;
 				str2display += TEXT("\" cannot be opened:\nFolder \"");
 				str2display += longFileDir;
-				str2display += TEXT("\" doesn't exist.");
-				::MessageBox(_pPublicInterface->getHSelf(), str2display.c_str(), TEXT("Cannot open file"), MB_OK);
+				str2display += TEXT("\" doesn't exist."); 

```

Comment:
New UI text should be made language translatable.

Reply:
How is translation handled in Notepad++? 
What would I need to add in order to enable localization?
Please let me know and I will fix it.
Thanks!

Reply:
What you need is `NativeLangSpeaker`, see https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/src/localization.h.
Take a look at other places that are already localized for examples.

Reply:
Perfect, I'll get the localization work done tonight and get a fix in. Thanks for the help!